### DISCUSSION TITLE
Q-popup proxy on one specific q-select value

### DISCUSSION THREAD

**Discussion by dost-mmg** [quasarframework/quasar#12813] [General - Components / Directives / etc]
I would like q-date to appear only when 'custom' is selected on a q-select menu. Right now, the popup proxy appears only after clicking again on the q-select value after selection. 

Separately, is there a way to trigger the popup proxy via method?

Here is the codepen: https://codepen.io/kiggs1881/pen/gOXXNqG?editors=1111

**Comment by MilosPaunovic** [comment]
Codepen link is broken.

**Reply by dost-mmg** [reply]
Shoot. Apologies. Updated: https://codepen.io/kiggs1881/pen/gOXXNqG?editors=1111
